exec/java-exec/src/test/java/org/apache/drill/exec/store/log/TestLogReader.java

```diff
@@ -177,6 +180,15 @@ private static LogFormatConfig firewallConfig() {
         regex, "ssdlog", null, schema);
   }
 
+  // DRILL-7853
+  private static LogFormatConfig issue7853Config() {
+    String regex = "([^ ]*) ([^ ]*) ([^ ]*) ([^ ]*):([0-9]*) ([^ ]*)[:-]([0-9]*) ([-.0-9]*) ([-.0-9]*) ([-.0-9]*) (|[-0-9]*) (-|[-0-9]*) ([-0-9]*) ([-0-9]*) \\\"([^ ]*) ([^ ]*) (- |[^ ]*)\\\" \\\"([^\\\"]*)\\\" ([A-Z0-9-]+) ([A-Za-z0-9.-]*) ([^ ]*) \\\"([^\\\"]*)\\\" \\\"([^\\\"]*)\\\" \\\"([^\\\"]*)\\\" ([-.0-9]*) ([^ ]*) \\\"([^\\\"]*)\\\"($| \\\"[^ ]*\\\")(.*)";
+    List<LogFormatField> schema = Lists.newArrayList(
+        new LogFormatField("type", "VARCHAR"),
+        new LogFormatField("time", "TIMESTAMP", "yyyy-MM-dd''T''HH:mm:ss.SSSSSSZ"));
```

Comment:
The original source of the error is likely that the user's log format included a Joda-specific time format that must be changed to use a Java time format. Makes sense to better report the error. I suppose we should also add this change to the docs. I had forgotten that the log reader allowed Joda formats. No tests caught this issue. Thanks for adding this test.